xgi/drawing/draw.py

```diff
@@ -1608,47 +1591,87 @@ def draw_dihypergraph(
+        assume the colors are specified in the same order as the hyperedges are found
+        in DH.edges. If None (default), use the size of the hyperedges.
+    dyad_lw : int, float, dict, iterable, optional
+        Line width of the bipartite edges. If int or float, use the same width for
+        all hyperedges. If a dict, must contain (hyperedge_id: width) pairs. If other
+        iterable, assume the widths are specified in the same order as the hyperedges
+        are found in DH.edges. By default, 1.
     node_labels : bool or dict, optional
         If True, draw ids on the nodes. If a dict, must contain (node_id: label) pairs.
         By default, False.
     hyperedge_labels : bool or dict, optional
         If True, draw ids on the hyperedges. If a dict, must contain (edge_id: label)
         pairs.  By default, False.
+    max_order : int, optional
+        Maximum of hyperedges to plot. If None (default), plots all orders.
+    arrowsize : int (default=10)
+        Size of the arrow head's length and width. See `matplotlib.patches.FancyArrowPatch`
+        for attribute `mutation_scale` for more info.
+    arrowstyle : str, optional
+        By default: '-\|>'. See `matplotlib.patches.ArrowStyle` for more options.
+    connectionstyle : string (default="arc3")
+        Pass the connectionstyle parameter to create curved arc of rounding
+        radius rad. For example, connectionstyle='arc3,rad=0.2'.
+        See `matplotlib.patches.ConnectionStyle` and
+        `matplotlib.patches.FancyArrowPatch` for more info.
     rescale_sizes: bool, optional
         If True, linearly interpolate `node_size` and between min/max values
         that can be changed in the other argument `params`.
         If `node_size` is a single value, this is ignored. By default, True.
-    iterations : int, optional
-        Maximum number of iterations taken to recompute the layout.
-        An original partial layout is computed solely based on the original nodes.
-        A full initial layout is then computed by simply adding the "edge" nodes
-        at the barycenters. This initial layout may suffer from overlap between edge-nodes.
-        After that, a spring layout is ran starting from the
-        initial layout, and each iteration makes all of the nodes overlap less.
     **kwargs : optional args
         Alternate default values. Values that can be overwritten are the following:
-        * "min_node_size" (default: 5)
-        * "max_node_size" (default: 30)
-        * "min_node_lw" (default: 0)
-        * "max_node_lw" (default: 5)
-        * "min_lines_lw" (default: 0)
-        * "max_lines_lw" (default: 50)
-        * "min_source_margin" (default: 0)
-        * "min_target_margin" (default: 0)
+        * min_node_size
+        * max_node_size
+        * min_node_lw
+        * max_node_lw
+        * min_edge_marker_size
+        * max_edge_marker_size
+        * min_edge_marker_lw
+        * max_edge_marker_lw
+        * node_fc_cmap
```

Comment:
not needed here, already a input argument

Reply:
Fixed.